### DISCUSSION TITLE
Distinguish adverbs of manner in POS tagging.

### DISCUSSION THREAD

**Discussion by kamihamiha123** [explosion/spaCy#2766] [New Features & Project Ideas]
## Feature description
Adverbs modify verbs to specify different information (time, manner, place or direction ) of the event described by the verb . It would be a great feature if spacy provides a way to distinguish each category of **adverbs** in POS tagging.

## Could the feature be a [custom component](https://spacy.io/usage/processing-pipelines#custom-components) or [spaCy plugin](https://spacy.io/universe)?
 No, Just extend tagging system to be more detailed

**Comment by kamihamiha123** [comment]
@ines is this possible, or it's beyond NLP state of art ?

**Comment by DuyguA** [comment]
I didn't quite understand the issue. Would it be possible to provide some examples?

**Comment by kamihamiha123** [comment]
@DuyguA  I am talking about the possibility to have a different tag for each of these adverb types :

**-  Adverbs of manner**
quickly
kindly

**- Adverbs of degree**
very
rather

**- Adverbs of frequency**
often
sometimes

**- Adverbs of time**
now
today

**- Adverbs of place**
here
nowhere

NB: We can consider also this enhancement for adjectives.

**Comment by DuyguA** [comment]
Sorry for the late answer.

POS tags are syntactical tags. These 'tags' that you mentioned are related to semantics. My opinion is that, such tags are not very suitable for POS tagging system but if semantical tags are introduced somehow why not...

p.s. Adjectives are not that straightforward. Consider for example:

That price was really **low**. //cheap
I'm in a **low** mood since months. //depressed
I was offered a **lower** position than I applied. //junior

Here's output from Wordnet about **low**:

```
Noun

    S: (n) low, depression (an air mass of lower pressure; often brings precipitation)
    S: (n) Low, David Low, Sir David Low, Sir David Alexander Cecil Low (British political cartoonist (born in New Zealand) who created the character Colonel Blimp (1891-1963))
    S: (n) low (a low level or position or degree)
    S: (n) first gear, first, low gear, low (the lowest forward gear ratio in the gear box of a motor vehicle; used to start a car moving)

Verb

    S: (v) moo, low (make a low noise, characteristic of bovines)

Adjective

    S: (adj) low (less than normal in degree or intensity or amount)
    S: (adj) low (literal meanings; being at or having a relatively small elevation or upward extension)
    S: (adj) low, low-toned (very low in volume)
    S: (adj) low (unrefined in character)
    S: (adj) low, low-pitched (used of sounds and voices; low in pitch or frequency)
    S: (adj) abject, low, low-down, miserable, scummy, scurvy (of the most contemptible kind)
    S: (adj) humble, low, lowly, modest, small (low or inferior in station or quality)
    S: (adj) depleted, low (no longer sufficient)
    S: (adj) broken, crushed, humbled, humiliated, low (subdued or brought low in condition or status)
    S: (adj) gloomy, grim, blue, depressed, dispirited, down, downcast, downhearted, down in the mouth, low, low-spirited (filled with melancholy and despondency)

Adverb

    S: (adv) low (in a low position; near the ground)
```

To summarize: Adverbs are not so difficult to semantically classify. However noun and adjective meanings can vary by the context, not so straightforward.

**Comment by kamihamiha123** [comment]
@DuyguA  What prevents Spacy to include adverb semantic tagging ?

**Comment by dshgna** [comment]
Came across this while Googling. Were there any further developments? 

This would potentially be a very useful feature. While working with topic models, I find that adverbs of manner are generally informative (e.g. disproportionately, legally, culturally, mentally) while the other types of adverbs (e.g. just, like) are mostly noise.
